checkpoint/keeper.go

```diff
@@ -116,10 +116,10 @@ func (k *Keeper) addCheckpoint(ctx sdk.Context, key []byte, checkpoint hmTypes.C
 	return nil
 }
 
-// GetCheckpointByIndex to get checkpoint by header block index 10,000 ,20,000 and so on
-func (k *Keeper) GetCheckpointByIndex(ctx sdk.Context, headerIndex uint64) (hmTypes.Checkpoint, error) {
+// GetCheckpointByNumber to get checkpoint by checkpoint number
+func (k *Keeper) GetCheckpointByNumber(ctx sdk.Context, number uint64) (hmTypes.Checkpoint, error) {
 	store := ctx.KVStore(k.storeKey)
-	headerKey := GetHeaderKey(headerIndex)
+	headerKey := GetHeaderKey(number)
```

Comment:
```suggestion
	headerKey := GetHeaderKey(number)
```
	checkpointKey := GetCheckpointKey(number)